tests/scripts/ci.py

```diff
@@ -77,6 +81,11 @@ def cmd(commands: List[Any], **kwargs: Any):
     return proc
 
 
+def get_build_dir(type: str) -> str:
```

Comment:
should we key off image name instead of type? probably not generally valid to take one build to another img

Reply:
it already is based on the image in the actual usages, changed the name to make it more clear